### PR TITLE
Charts: update valueErrorBar and customizeTooltip

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/BaseChart/1 Configuration/tooltip/customizeTooltip.md [hunk 1]
```diff
@@ -197,6 +197,110 @@ The format of the string is the following: "h: %highValueText% o: %openValueText
     </tr>
 </table>
```
Comment (albertov05) [inline api-reference/10 UI Components/BaseChart/1 Configuration/tooltip/customizeTooltip.md:199]: Can you apply the following changes above?

Line 45:  
What do you mean by "the similar type"? Is the type _the same_? For example:  
`In this instance, <b>argument</b> has the same type as the argument axis.`

Line 69:  
"folllowing" is misspelled. Please fix.

Line 82:  
`<td>The percent value of the point being hovered over.`

This description is exactly the same as the one on line 78. Is this intended?  
Also note that the closing **td** tag is missing.

Line 101:  
Please update this description to the following:  
`The first range value of the point being hovered over as it is set in the data source.`

Line 113:  
Please update this to the following:  
`The second range value of the point being hovered over as it is set in the data source.`

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/valueErrorBar/valueErrorBar.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ Error bars are used on charts to indicate an error or an uncertainty in a report
...
+Error bars can be generated either from concrete or calculated values. To generate one error bar, two values, high and low, are needed. If the [data source](/api-reference/10%20UI%20Components/BaseChart/1%20Configuration/dataSource.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource') of your chart provides concrete high and low values, assign the required data source fields to the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/#lowValueField) properties. To hide individual error bars, set the corresponding low and high values in the data source to `null`.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/valueErrorBar/valueErrorBar.md]:
```suggestion
Error bars can be generated either from concrete or calculated values. To generate one error bar, two values, high and low, are needed. If the chart [data source](/api-reference/10%20UI%20Components/BaseChart/1%20Configuration/dataSource.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource') supplies concrete high and low values, assign the required data source fields to the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/#lowValueField) properties. To hide individual error bars, set the corresponding low and high values in the data source to `null`.
```

File: api-reference/10 UI Components/BaseChart/1 Configuration/tooltip/customizeTooltip.md (hunk 1 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/BaseChart/1 Configuration/tooltip/customizeTooltip.md:199]: Done

File: api-reference/10 UI Components/BaseChart/1 Configuration/tooltip/customizeTooltip.md [hunk 3]
```diff
@@ -75,11 +75,11 @@ The folllowing **pointInfo** fields are available for the stacked-like series su
     </tr>
     <tr>
         <td><code>percent</code></td>    
-        <td>The percent value of the point being hovered over.</td>
+        <td>The percent value of the point being hovered over in fractions. For example, <code>0.6</code>.</td>
```
Comment (albertov05) [inline api-reference/10 UI Components/BaseChart/1 Configuration/tooltip/customizeTooltip.md]:
```suggestion
        <td>The percent value of the point being hovered over in decimal format. For example, <code>0.6</code>.</td>
```

**Pull request merged**